docs/usage.md

```diff
@@ -26,28 +26,28 @@ run on a build platform, this encapsulation can guarantee that the platform has
 dependencies needed for the build. One class of hermetic build implementations is to restrict external network access
 during the build itself, requiring that all dependencies are declared and pre-fetched before the build occurs.
 
-In order to support this class of hermetic builds, not only does Cachi2 need to pre-fetch the dependencies, but some
+In order to support this class of hermetic builds, not only does Hermeto need to pre-fetch the dependencies, but some
 build flows will need additional changes (i.e. leveraging defined [environment variables](#generate-environment-variables)
-or using Cachi2 to [inject project files](#inject-project-files)).
+or using Hermeto to [inject project files](#inject-project-files)).
 
 ### Pre-fetch dependencies
 
 The first step in creating hermetic builds is to fetch the dependencies for one of the [supported package managers](../README.md#package-managers).
 
 ```shell
-cachi2 fetch-deps \
+hermeto fetch-deps \
   --source ./foo \
-  --output ./cachi2-output \
+  --output ./hermeto-output \
   --sbom-output-type cyclonedx \
   '{"path": ".", "type": "<supported package manager>"}'
 ```
 
-* `--source`           - the path to a *git repository* on the local disk `[default: .]`
```

Comment:
Did you drop defaults intentionally?

Reply:
The mundane nature of these changes makes it extra difficult for the reviewer to spot these types of errors, so kudos!